src/components/AppNavigation/Settings/ShortcutOverview.vue

```diff
@@ -0,0 +1,116 @@
+<!--
+  - @copyright Copyright (c) 2020 Georg Ehrke <oc.list@georgehrke.com>
+  - @author Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<template>
+	<Modal
+		class="shortcut-overview-modal"
+		size="middle"
+		:title="$t('calendar', 'Shortcut overview')"
+		@close="$emit('close')">
+		<section
+			v-for="category in shortcuts"
+			:key="category.categoryId"
+			class="shortcut-section">
+			<h3 class="shortcut-section__header">
+				{{ category.categoryLabel }}
+			</h3>
+			<div
+				v-for="(shortcut, index) in category.shortcuts"
+				:key="index"
+				class="shortcut-section-item">
+				<span class="shortcut-section-item__keys">
+					<template
+						v-for="(keyCombination, index2) of shortcut.keys"
+						class="shortcut-section-item__key-combination">
+						<template v-for="(key, index3) in keyCombination">
+							<kbd :key="index3">{{ key }}</kbd>
+							<span
+								v-if="index3 !== (keyCombination.length - 1)"
+								:key="index3"
```

Comment:
```
Duplicate keys detected: '0'. This may cause an update error.
```
Probably
> Children of the same common parent must have unique keys. Duplicate keys will cause render errors.

Possible fix:
```js
:key="`${index}${index2}${index3}`"
```